contracts/ERC1238/ERC1238.sol

```diff
@@ -85,6 +83,26 @@ contract ERC1238 is IERC1238 {
         return batchBalances;
     }
 
+    function getMintApprovalMessageHash(
+        address to,
+        uint256 id,
+        uint256 amount
+    ) public view override returns (bytes32) {
+        return keccak256(abi.encode(to, id, amount, block.chainid, address(this)));
+    }
+
+    function getMintApprovalMessageHash(
```

Comment:
Isn't this `getMintBatchApprovalMessageHash`?

Reply:
Fixed